deployment/common/view/nops.go

```diff
@@ -5,13 +5,13 @@ import (
 	"fmt"
 
 	nodev1 "github.com/smartcontractkit/chainlink-protos/job-distributor/v1/node"
+
 	"github.com/smartcontractkit/chainlink/deployment"
 )
 
 type NopView struct {
 	// NodeID is the unique identifier of the node
 	NodeID       string                `json:"nodeID"`
-	PeerID       string                `json:"peerID"`
```

Comment:
hmm was this a bug? I don't think we should be removing it...

Reply:
I did not mean to remove that at all... maybe a mistake on save? will add it back